server/docs/hellobooks_api_doc.yaml

```diff
@@ -552,21 +570,15 @@ paths:
                   type: number
                 damaged:
                   type: string
```

Comment:
Isn't this a boolean?

Reply:
I implemented it as a string

Reply:
I see. That wasn't the best option but could pass